src/js/edu-benefits/5490/config/form.js

```diff
@@ -73,7 +75,14 @@ const formConfig = {
   chapters: {
     applicantInformation: {
       title: 'Applicant Information',
-      pages: {}
+      pages: {
+        applicantInformation: applicantInformation([
+          'relativeFullName',
```

Comment:
I didn't refresh between my first and second comments, so I didn't see your comment, @goldenmeanie. It was a bit redundant, sorry.

Yeah, it is split up like that. `relativeFullName` is a sort of container for all the pieces.